secrets-provider/aws/index.ts

```diff
@@ -10,12 +10,36 @@ const config = new pulumi.Config();
 const bucketName = config.require('bucketName');
 const secretValue = config.requireSecret('secretValue');
 
+export function configureACL(bucketName: string, bucket: aws.s3.BucketV2, acl: string): aws.s3.BucketAclV2 {
```

Comment:
For this example I don't think we need to do anything with the ACL. By default ACLs are disabled and block public access is turned on for new buckets. Since this looks to be a normal bucket (not a public website bucket) I think the default is what we want.